Symfony/CS/Fixer/Symfony/PhpdocScalarFixer.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Symfony;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\DocBlock\DocBlock;
+use Symfony\CS\DocBlock\Line;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Graham Campbell <graham@mineuk.com>
+ */
+class PhpdocScalarFixer extends AbstractFixer
+{
+    /**
+     * The annotation tags search inside.
+     *
+     * @var array
+     */
+    private static $tags = array('param', 'return', 'type', 'var');
+
+    /**
+     * The types to fix.
+     *
+     * @var array
+     */
+    private static $types = array(
+        'integer' => 'int',
+        'boolean' => 'bool',
+        'real'    => 'float',
+        'double'  => 'float',
+    );
+
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        foreach ($tokens->findGivenKind(T_DOC_COMMENT) as $token) {
+            $doc = new DocBlock($token->getContent());
+
+            foreach ($doc->getAnnotationsOfType(self::$tags) as $annotation) {
+                $this->fixScalars($doc->getLine($annotation->getStart()), $annotation->getTag()->getName());
+            }
+
+            $token->setContent($doc->getContent());
+        }
+
+        return $tokens->generateCode();
+    }
+
+    /**
+     * Fix scalar types.
+     *
+     * We must be super careful not to modify parts of words.
+     *
+     * @param Line   $line
+     * @param string $tag
+     */
+    private function fixScalars(Line $line, $tag)
+    {
+        $content = $line->getContent();
+        $tagSplit = preg_split('/ *\@'.$tag.' */', $content);
```

Comment:
I always thought ` *` would be "start with space, then find anything up until @`?